README.md

```diff
@@ -1,44 +1,25 @@
 # VA.gov Team Repository
-*Last updated: [9/17/2019](#document-history)*
 
 This project aims to improve the lives of Veterans, their families, and their caregivers by facilitating the creation of interactive experiences on VA.gov on par with the private sector. It's comprised of a platform (the Veteran-facing Services Platform, aka VSP) which serves as the common framework for building cohesive products on VA.gov, and the VA.gov products themselves (the Veteran Facing Services, aka VFSs).
 
-This repository is used to store *non-code* documents and files created and used by teams building the VSP and VFSs.
+This repository is used to store *non-code* documents and files created and used by teams building the VSP and VFSs. It is also used for issue tracking for many parts of the VSP platform.
```

Comment:
The va.gov-team repo is used for issue tracking across VSP and VFS, so this could say something like: "It is also used for issue tracking across the platform and teams building on VA.gov."

Reply:
That sounds good. Will make that change.